Havit.Blazor.Components.Web.Bootstrap/Forms/Autosuggests/Internal/HxAutosuggestInternal.razor.cs

```diff
@@ -225,6 +225,7 @@ private async Task HandleInputEnterKeyDown()
 			if (HighlightFirstSuggestionEffective)
 			{
 				await HandleItemClick(suggestions.FirstOrDefault());
+				await DestroyDropdownAsync();
```

Comment:
The event-callback can be long-running task, we should close the dropdown first and then call the callback?
@Harvey1214  Will fix on my own by exchanging those two lines (if you see any troubles with that, let me know).